Competition2017/tests/competition/subsystems/collector/CollectorSubsystemTest.java

```diff
@@ -0,0 +1,30 @@
+package competition.subsystems.collector;
+
+import org.junit.Test;
+
+
+public class CollectorSubsystemTest extends CollectorTestBase {
+    
+   
+    @Test
+    public void stopTest(){
+        collect.stop();
+        
+        verifyCollectSetpoints(0);
```

Comment:
I chose 0 because the power of the Collector Motor should be 0 when the stop command is being command.